content/zh/docs/concepts/instrumentation/code-based.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 基于代码的方式
+description: 了解设置基于代码插桩的基本步骤
+weight: 20
+aliases: [manual]
+default_lang_commit: deb98d0648c4833d9e9d77d42e91e2872658b50c
+---
+
+## 导入 OpenTelemetry API 和 SDK {#import-the-opentelemetry-api-and-sdk}
+
+你首先需要在服务代码中导入 OpenTelemetry。如果你正在开发一个库或其他打算被可运行二进制文件使用的组件，
+那么你只需要依赖 API。如果你的构件是一个独立的进程或服务，那么你需要同时依赖 API 和 SDK。
+有关 OpenTelemetry API 和 SDK 的更多信息，请参阅[规范](/docs/specs/otel/)。
+
+## 配置 OpenTelemetry API {#configure-the-opentelemetry-api}
+
+为了创建链路或指标，你需要先创建一个 tracer 和/或 meter 提供程序。通常，我们建议
```

Comment:
Why isn't logger mentioned here? Maybe the English version of the doc is outdated.

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/api.md#logger

Reply:
Probably, need to revise the original english text.